tfx/components/example_validator/component.py

```diff
@@ -78,6 +80,8 @@ def __init__(self,
         should contain at least 'eval' split. Other splits are currently
         ignored.
       schema: A Channel of type `standard_artifacts.Schema`. _required_
+      exclude_splits: Names of splits that the example validator should not
+        validate.
```

Comment:
I think we need a default value for it, so existing user don't need to set it to have the current behavior

Reply:
yes, added the default behaviors